lib/logger.js

```diff
@@ -121,6 +121,29 @@ var streamConverter = {
 };
 var streamConverterList = Object.keys(streamConverter);
 
+Logger.prototype._addErrorHandler = function() {
+    var self = this;
+    self._logger.on('error', function(err, failedStream) {
+        // If some fatal error occurred in one of the logger streams,
+        // we can't do much, so ignore.
+
+        if (err.code === 'ENOSPC') {
```

Comment:
@Pchelolo: For remote UDP streams temporary failures are more likely, and we know that those don't cause memory issues. Lets perhaps limit disabling streams to file & stdout streams, or anything-but-gelf?